lib/main.dart

```diff
@@ -62,6 +66,26 @@ void main() async {
   BackgroundFetch.registerHeadlessTask(backgroundFetchHeadlessTask);
 }
 
+Future _deleteData() async {
```

Comment:
I think it is a bit too much to delete ALL preferences, as it will mean that the user will have to set back again home position, notifications, and we don't warn him we wiped everything.

If the issue here is about the news feature, I propose to remove only a list of prefs passed as parameters instead of all.

Reply:
Ok, I'll rework this

Reply:
I really have no clue which local data makes the application bug. Users only see 2-3 walks per date, some say when they filter on home or position, they don't see the distances anymore.

What we know:
1) Clearing the cache and storage makes it work again.
2) Users try and uninstall the application, but that doesn't work, cache/storage isn't deleted.
3) In the quick bugfix I released to the store, I fixed the clear news_bug, and I also updated the database version so that the database would be recreated and the walks refetched. Apparently that didn't fix the problem, users still only saw a few walks even after upgrading to the latest bugfix release.
4) The ANR's and Crashed we get at the moment are 95% from the bugged release, it seems that with the new bugfix release, the application crashes way less.
5) I've release only the bugfixed release on IOS and I haven't gotten any complaints of crashes yet and from the analytics, it doesn't seem like there are any more crashes then previous releases.

The question is: Which local data do we want to delete. I'm inclined to also take home and filter data into the delete, as users experience problems with that and we can't pinpoint exactly which local data is the problem

Reply:
I suggest to try without the news feature: that’s the last major change we did. The issue was absent before that. So pretty sure that emptying prefs just treats the symptoms, not the illness

Reply:
Maybe we should investigate the way Android backups app data if it is activated, and ask users if it is enabled - although pretty sure they don’t know the answer to that.

https://developer.android.com/guide/topics/data/autobackup